README.md

```diff
@@ -20,7 +20,7 @@ Or install it yourself as:
 
 ### Prerequisites
 
-Ensure you have *GraphViz* installed. On macOS you can install it using Homebrew with the following command:
```

Comment:
This was right the first time. [macOS](https://www.apple.com/macos/high-sierra/) with a lowercase "m" is how Apple spells it.

Reply:
👍  fixed